### PR TITLE
Adding Organization movements view same functionality like users movements view

### PR DESCRIPTION
**Pull request by sseerrggii**
Changes:

 - use new `_movements` partial in both user and organization detail DRY
 - pagination in both user and organization movements list
 - do not show movements list in organization details if not logged in
 - add links to user names in movements lists

### CODE REVIEW

File: app/views/organizations/show.html.haml [hunk 1]
```diff
@@ -81,10 +80,29 @@
...
           %tr
-            %td= l mv.created_at, format: :long
-            %td= mv.other_side.account
-            %td= mv.transfer.post
-            %td= mv.transfer.reason
-            %td= seconds_to_hm(mv.amount)
\ No newline at end of file
+            %td= l mv.created_at.to_date, format: :long
+            %td
+              - mv.other_side.account.tap do |account|
```
Comment (rewritten) [inline app/views/organizations/show.html.haml]: Nice way to avoid local variable assignment 😄 👍

**Comment by rewritten** [comment]
👏 👏 👏 @sseerrggii but please fix style.

**Comment by sseerrggii** [comment]
Ok @rewritten !!!

File: app/controllers/organizations_controller.rb [hunk 2]
```diff
@@ -18,6 +18,7 @@ def index
   end
 
   def show
+  @movements = @organization.account.movements.order("created_at DESC").page(params[:page]).per(10)
```
Comment (enricostano) [inline app/controllers/organizations_controller.rb]: Could you please do a `organization.account.movements.order("created_at DESC").page(params[:page]).per(10).explain` in Rails console and paste the generated query here? I think that we can optimize the query.

Comment (sseerrggii) [inline app/controllers/organizations_controller.rb]: Account Load (136.6ms)  SELECT  "accounts".* FROM "accounts" WHERE "accounts"."accountable_id" = $1 AND "accounts"."accountable_type" = $2 LIMIT 1  [["accountable_id", 2], ["accountable_type", "Organization"]]
 
Account Load (136.6ms)  SELECT  "accounts".* FROM "accounts" WHERE "accounts"."accountable_id" = $1 AND "accounts"."accountable_type" = $2 LIMIT 1  [["accountable_id", 2], ["accountable_type", "Organization"]]

Comment (sseerrggii) [inline app/controllers/organizations_controller.rb]: Este es el bueno  :+1: 
Movement Load (160.6ms)  SELECT  "movements".* FROM "movements" WHERE "movements"."account_id" = $1  ORDER BY created_at DESC LIMIT 10 OFFSET 0  [["account_id", 2]]
  Movement Load (160.6ms)  SELECT  "movements".* FROM "movements" WHERE "movements"."account_id" = $1  ORDER BY created_at DESC LIMIT 10 OFFSET 0  [["account_id", 2]]
=> EXPLAIN for: SELECT  "movements".* FROM "movements" WHERE "movements"."account_id" = $1  ORDER BY created_at DESC LIMIT 10 OFFSET 0 [["account_id", 2]]
                                               QUERY PLAN
--------------------------------------------------------------------------------------------------------
 Limit  (cost=14.45..14.47 rows=7 width=32)
   ->  Sort  (cost=14.45..14.47 rows=7 width=32)
         Sort Key: created_at
         ->  Bitmap Heap Scan on movements  (cost=4.21..14.35 rows=7 width=32)
               Recheck Cond: (account_id = 2)
               ->  Bitmap Index Scan on index_movements_on_account_id  (cost=0.00..4.21 rows=7 width=0)
                     Index Cond: (account_id = 2)
(7 rows)

Comment (enricostano) [inline app/controllers/organizations_controller.rb]:
```
organization
  .account
  .movements
  .order("created_at DESC")
  .page(params[:page])
  .per(10)
```

**Comment by sseerrggii** [comment]
Can you help me with missing style to improve, hating hound :sob:

File: app/views/organizations/show.html.haml [hunk 3]
```diff
@@ -81,10 +80,29 @@
...
+                    = link_to account,
+                              organization_path(account.accountable)
+                  - elsif account.accountable.active
+                    = link_to account.accountable.display_name_with_uid,
+                              user_path(account.accountable.user)
+                  - else
+                    = t(".inactive_user")
+                - else
+                  = t(".deleted_user")
+            %td
```
Comment (rewritten) [inline app/views/organizations/show.html.haml]:
```haml
- if mv.transfer.try(:post).try(:active?)
  = link_to(mv.transfer.post, offer_path(mv.transfer.post))
- else
  = mv.transfer.post
```

Comment (sseerrggii) [inline app/views/organizations/show.html.haml]: done! Thanks!

File: app/views/partials/_movements.html.haml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                = link_to(mv.transfer.post, offer_path(mv.transfer.post))
+              - else
+                = mv.transfer.post
+            %td
+              = mv.transfer.reason
+              - if mv.transfer.reason.present? && admin?
+                - op = { method: :put, data: { confirm: t(".delete_reason") } }
+                = link_to(delete_reason_transfer_path(mv.transfer), op) do
+                  %span.glyphicon.glyphicon-trash
+            %td= seconds_to_hm(mv.amount.abs)
```
Comment (markets) [inline app/views/partials/_movements.html.haml]: @sseerrggii are you sure about this? Do we want to show always positive amounts?

Nice PR by the way! 👏 👍

Comment (sseerrggii) [inline app/views/partials/_movements.html.haml]: @markets You are **abs**olutely right! Thanks!

**Comment by enricostano** [comment]
Move `_movements.html.haml` to `app/views/shared` and :+1: 

Vamooooss :clap: :clap: :clap:

**Comment by rewritten** [comment]
Good!

Next step would be either to compose two views (one for @organization/@user and the other for @movements) or to pass a local to the partial, delegating the specifics of "which" movements to render to something else, not the controller.

**Pull request merged**